superset/assets/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -108,15 +109,16 @@ class ChartHolder extends React.Component {
       onResizeStop,
       handleComponentDrop,
       editMode,
+      isModalSlice,
     } = this.props;
 
     // inherit the size of parent columns
     const widthMultiple =
       parentComponent.type === COLUMN_TYPE
         ? parentComponent.meta.width || GRID_MIN_COLUMN_COUNT
         : component.meta.width || GRID_MIN_COLUMN_COUNT;
-
     return (
+      (!isModalSlice || editMode) && (
```

Comment:
we can remove (!isModalSlice || editMode) this condition and no need to pass this props in charHolder. We can check DashboardComponent.jsx container component should rendered or not. After these changes, we stop unwanted default query execution for popup slice on dashboard load.

```
class DashboardComponent extends React.PureComponent {
  render() {
    const { component, isModalSlice, editMode } = this.props;
    const Component = component ? ComponentLookup[component.type] : null;
    return Component && (!isModalSlice || editMode) ? <Component {...this.props} /> : null; 
  }
} ```

Reply:
done